common/src/main/java/com/lightstep/tracer/shared/Options.java

```diff
@@ -54,6 +54,15 @@
             }}
     );
 
+    /**
+     * Select the client provider explicitly.
+     * This is only needed if both client providers are in the classpath.
+     */
+    public enum CollectorClient {
+        HTTP,
+        GRPC
```

Comment:
I'd like to have an `ANY` value here, so we don't show/log any error in case one of them was found successfully.

This is to keep the current use-case, where we simply decide what we find.

Reply:
you'd get the `ANY` behaviour if you don't call `withCollectorClient`